source/2_platform/rex_directx/include/rex_directx/directx_feature_shader_model.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "rex_engine/types.h"
+#include "rex_std/string.h"
+
+struct ID3D12Device;
+
+namespace rex
+{
+  struct ShaderModelInfo
+  {
```

Comment:
the same comment applies as the directx feature level.
I don't think this struct is needed, we have enum reflection to get the name and the int represenation hides what it's supposed to be and will only add complexity